Tutorials/index.md

```diff
@@ -1,253 +1,32 @@
-            we strongly recommend you read it before starting
-            the tutorials.
-  - `projects/sel4-tutorials/docs/seL4-APILib-details.pdf`: This is the actual tutorial.
-
-### CAmkES tutorials
-
-These tutorials get you started with our component system CAmkES, which
-allows you to configure static systems through components. CAmkES
-generates the glue code for interacting with seL4 and is designed for building
-high-assurance, static systems.
-
-These tutorials work similarly to the SEL4 tutorials in that they are
-guided by a slide presentation. There are half-completed sample
-applications, with a set of slides giving instructions, with TASK
-challenges once again. There are also completed sample solutions.
-
-{%- for t in tutorials %}
-{%- if t.tutorial-order contains 'camkes' %}
-1. [{{t.title}}]({{t.url}}) {{t.description}}
-{%- endif %}
-{%- endfor %}
-
-- The slide presentations to guide you through the tutorials are
-        in this
-        file: `projects/sel4-tutorials/docs/CAmkESTutorial.pdf`.
-
-### Virtual machines
-
-These tutorials show how to run Linux guests on Camkes and how to communicate between them.
-Currently these tutorials are for x86 only.
-
-{%- for t in tutorials %}
-{%- if t.tutorial-order contains 'vm' %}
-1. [{{t.title}}]({{t.url}}) {{t.description}}
-{%- endif %}
-{%- endfor %}
-
-### MCS extensions
-
-The MCS extensions are upcoming API changes to seL4.
-
-{%- for t in tutorials %}
-{%- if t.tutorial-order contains 'mcs' %}
-1. [{{t.title}}]({{t.url}}) {{t.description}}
-{%- endif %}
-{%- endfor %}
-
-## What next?
-
-You can try building and running [seL4test](../seL4Test).
-
-Next steps include working on one of our [suggested
-projects](/SuggestedProjects.html) or helping to expand the collection
-of [libraries and
-components](/projects/available-user-components.html) available to
-build seL4-based systems.
+<h2>Categories</h2>
+The tutorials are split into a number of broad categories:
+
+- [About seL4](GettingStarted/about-seL4) and [Getting started with the Microkit tutorial](GettingStarted/microkit) introduce seL4 concepts.
+- The [seL4 Kernel tutorials](seL4Kernel/overview.md) are a deep dive into seL4 concepts.
+- [MCS](MCS/mcs) introduces seL4 MCS extensions.
+- [Dynamic Libraries](DynamicLibraries/dynamic-1) covers the libraries that have been developed for rapidly prototyping systems on seL4.
+- [CAmkES](CAmkES/hello-camkes-0) is a platform for building componentised systems for embedded platforms.
+- [Microkit](https://trustworthy.systems/projects/microkit/tutorial/)is an operating system framework on top of seL4 provides a small set of simple abstractions that ease the design and implementation of statically structured systems on seL4. (Links to the same tutorial as in the [Getting Started](GettingStarted/microkit) section.)
+- [Rust](https://github.com/seL4/rust-sel4) provide crates for supporting the use of Rust in seL4 userspace.
```

Comment:
I did not come across this content when I was navigating the new site -- which page does it get rendered to?

Reply:
This page was rendered when the user clicked on "Tutorials." I've now redirected "Tutorials" to overview, and used much of the above content in the overview page.